doc.go

```diff
@@ -29,6 +36,9 @@
 //
 // There are runnable examples at
 // https://github.com/honeycombio/beeline-go/tree/master/examples and examples
-// of each wrapper in the dgodoc.
+// of each wrapper in the godoc.
 //
+// TODO create two comprehensive examples, one showing basic beeline use and the
```

Comment:
TODO?

Reply:
yup, will leave to last though, because examples can come after landing.